src/features/auth/components/Donation.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Button } from "components/ui/Button";
+import { InnerPanel, OuterPanel, Panel } from "components/ui/Panel";
+import upArrow from "assets/icons/arrow_up.png";
+import downArrow from "assets/icons/arrow_down.png";
+import question from "assets/icons/expression_confused.png";
+import leftArrow from "assets/icons/arrow_left.png";
+import rightArrow from "assets/icons/arrow_right.png";
+import { useLazyEffect } from "lib/utils/useLazyEffect";
```

Comment:
Looks like this hook is missing from this branch 🤔  I can't find a `lib/utils/useLazyEffect`